src/themes/teams/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+import { pxToRem } from '../../../lib'
+
+export const teamsPxToRem = (sizeInPx: number) => pxToRem(sizeInPx, 14)
```

Comment:
shouldn't `14` magic number be a property of the theme object, maybe `defaultFontSize`?

Reply:
### Util

Since this util is scoped to the teams theme, I propose not prefixing it with `teams`.  In fact, since the base util will need to use a _required_ key from the theme (html font size), it seems this util should remain a base util provided by Stardust itself so all consumers can rely on the correct font size being used.

### Theme based html font size

Yep, bump.  Reminder that this is required as the value needs to be configurable from the Provider.  This enables the same component to be used in apps with different html font sizes.

I would propose using `htmlFontSize` as the top level key, however.  Since, this is explicitly what the value is.  It is also used by the same name for the same purpose in [Material's theme config](https://material-ui.com/customization/themes/#typography---html-font-size).